server/historian/packages/historian-base/src/routes/utils.ts

```diff
@@ -54,7 +54,12 @@ export async function createGitService(
     }
 
     const details = await tenantService.getTenant(tenantId, token);
-    const service = new RestGitService(details.storage, cache);
+    const customData: ITenantCustomDataExternal = details.customData;
+    let writeToExternalStorage = false;
+    if (customData.externalStorageData != null) {
+        writeToExternalStorage = true;
+    }
```

Comment:
const writeToExternalStorage = customData.externalStorageData !== null ? true: false;

Also make sure to always use '==='/'!=='

Reply:
(nit) could just be `customData.externalStorageData !== null`